app/components/ExpirationInput/ExpirationInput.css.js

```diff
@@ -0,0 +1,40 @@
+import { StyleSheet } from 'react-native';
+import { Colors } from '../../theme/theme';
+
+export const ExpirationInputCSS = StyleSheet.create({
+  expirationInput: {
+    flexDirection: 'column',
+  },
+  expirationToggleRow: {
+    flexDirection: 'row',
+    paddingRight: 16,
+    justifyContent: 'space-between',
+    height: 48
+  },
+  icon: {
+    justifyContent: 'center',
+    marginLeft: 16,
+    marginRight: 32
+  },
+  switchLabel: {
+    flex: 1,
+    textAlignVertical: 'center',
+    fontSize: 16,
+    color: Colors.primaryTextBlack
+  },
+  switch: {
+
+  },
```

Comment:
I was planning on styling the toggle/switch on the expiration input. I was having difficulties with it though and must have ended up not applying any styles to it. I can remove it.